[2023-08-14T16:19:15.711+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: emrPipeline.s3_file_check manual__2023-08-14T16:18:38+00:00 [queued]>
[2023-08-14T16:19:15.750+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: emrPipeline.s3_file_check manual__2023-08-14T16:18:38+00:00 [queued]>
[2023-08-14T16:19:15.750+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 31
[2023-08-14T16:19:15.817+0000] {taskinstance.py:1327} INFO - Executing <Task(S3KeySensor): s3_file_check> on 2023-08-14 16:18:38+00:00
[2023-08-14T16:19:15.849+0000] {standard_task_runner.py:57} INFO - Started process 9973 to run task
[2023-08-14T16:19:15.925+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'emrPipeline', 's3_file_check', 'manual__2023-08-14T16:18:38+00:00', '--job-id', '1470', '--raw', '--subdir', 'DAGS_FOLDER/emr.py', '--cfg-path', '/tmp/tmpzyp51q71']
[2023-08-14T16:19:15.938+0000] {standard_task_runner.py:85} INFO - Job 1470: Subtask s3_file_check
[2023-08-14T16:19:16.073+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-14T16:19:16.423+0000] {task_command.py:410} INFO - Running <TaskInstance: emrPipeline.s3_file_check manual__2023-08-14T16:18:38+00:00 [running]> on host d4028c5ba2ac
[2023-08-14T16:19:17.210+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='emrPipeline' AIRFLOW_CTX_TASK_ID='s3_file_check' AIRFLOW_CTX_EXECUTION_DATE='2023-08-14T16:18:38+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-14T16:18:38+00:00'
[2023-08-14T16:19:17.212+0000] {s3.py:103} INFO - Poking for key : s3://myawsbucket19080/hiveStep/raw/tripdata.csv
[2023-08-14T16:19:17.325+0000] {base.py:73} INFO - Using connection ID 'aws_connection' for task execution.
[2023-08-14T16:19:17.325+0000] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_connection', conn_type='aws') credentials retrieved from login and password.
[2023-08-14T16:19:20.274+0000] {base.py:255} INFO - Success criteria met. Exiting.
[2023-08-14T16:19:20.342+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=emrPipeline, task_id=s3_file_check, execution_date=20230814T161838, start_date=20230814T161915, end_date=20230814T161920
[2023-08-14T16:19:20.690+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-14T16:19:21.097+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
